login.php

```diff
@@ -6,7 +6,7 @@
 
 $db = new PDO('mysql:host=127.0.0.1;dbname=users','root');
```

Comment:
maybe put in a function and place in other file to add to .gitignore and call $db = dbconnect func. here instead?